### DISCUSSION TITLE
Crazy swarm drone crash when using Skybrush live

### DISCUSSION THREAD

**Discussion by FlyPX24** [bitcraze/discussions#1251] [Q&A]
Hello everyone, 
We have a problem that we have been dealing with for the past 2 months so if there is any help it will be appreciated. We bought the crazy swarm with lighthouse bundle for out graduation project. We are facing the last problem where if its solved we will be done with the project (hopefully)

The problem:
1- When doing the set up we try our best to get the lighthouse base stations at the right angle but we couldn't figure out how to measure that angle or get it where both base stations beams can be perpendicular to each other.

2- We are using Blender with the Skybrush add-ons to create the choreography we want the swarm to shape. so far we are facing a problem where some drones somehow flip up side down and crash out of no where when we are using the Skybrush live software (the drones that crash are random). We also noticed that when we use cfclient to test the drone takeoff its more stable than when using takeoff on Skybrush live (we are not sure if it's because we are using 10 drones a

t ones or not). The first video you can see what I mean when the drone flips out of no where (3 drones flipped) the formation that they should do is making a spiral, "A", "U", "M", then land back as they was. The second video is where we tested the 3 drones that flipped alone but they didn't flip or have any problem.

Note that: We updated all the drones firmware including the lighthouse deck. We are welling to provide feedback later to make it easier for you to help us.

https://github.com/bitcraze/discussions/assets/164927472/6779d816-79c9-40b2-a29d-0a07bf1748ec

https://github.com/bitcraze/discussions/assets/164927472/07cf1e39-e95d-4e08-b42b-c0cf7992c8a9

**Comment by knmcguire** [comment]
So, there are two parts to your discussion. The lighthouse setup we can help out with but the skybrush support is not something we can do. So for question 2 I'll need to refer you to the support team at collmot https://skybrush.io/support/ 

> 1- When doing the set up we try our best to get the lighthouse base stations at the right angle but we couldn't figure out how to measure that angle or get it where both base stations beams can be perpendicular to each other.

This is unclear to us what you try to de here. Why do you need to measure or setup the angle? There should be an automatic geometry estimation method that already does this, so were you able to follow the tutorial for setting up the system here https://www.bitcraze.io/documentation/tutorials/getting-started-with-lighthouse/? Perhaps some more clarification of you issue or what you are trying to solve.

**Reply by FlyPX24** [reply]
Thank you for your reply, I simply meant to ask about how to make sure that the angle of the lighthouse base stations are making 90 degrees (as seen in the picture below). Because we are dealing with instability in our shows, and we have one of the lighthouses that is vibrating since we got it so we are trying to figure out if its because of the vibration or our setup is wrong.

I will contact Skybrush for the other part.
![90degree](https://github.com/bitcraze/discussions/assets/164927472/b035678d-87a0-455b-b3e9-1f422967a286)

**Reply by knmcguire** [reply]
ah like that. It doesn't have to be exact, and actually I think this only really holds for lighthouse basetation V1. it's just a guide line to get better coverage but you don't have to follow it to the point. As long as one Crazyflie is able to see at least one LH basestation at a time, then it should be able to fly
